### PR TITLE
Delete media

### PR DESCRIPTION
**Pull request by ri7nz**
i fix issues https://github.com/LevPasha/Instagram-API-python/issues/311
for delete media
this function is missing media_type parameters, so i add this parameter on my pull request

thanks @LevPasha 
regards, 
-- @ri7nz

### CODE REVIEW

File: InstagramAPI/InstagramAPI.py [hunk 1]
```diff
@@ -567,10 +568,11 @@ def mediaInfo(self, mediaId):
                            'media_id': mediaId})
         return self.SendRequest('media/' + str(mediaId) + '/info/', self.generateSignature(data))
 
-    def deleteMedia(self, mediaId):
+    def deleteMedia(self, mediaId, media_type=1):
```
Comment (ri7nz) [inline InstagramAPI/InstagramAPI.py]: media_type have 3 value type  { 1: 'Photo', '2': 'Video', '3': 'PhotoCard'} .

**Comment by ri7nz** [comment]
Thank you very much , sir @LevPasha 
I am very respectful, appreciative, and very happy to help and hopefully I can contribute again

regards, @ri7nz

**Comment by desecho** [comment]
@ri7nz Why do we need the media type here? Was it not deleting medias before?

**Comment by ri7nz** [comment]
@desecho before , this library can't delete media , check this issue https://github.com/LevPasha/Instagram-API-python/issues/311 
need media type before send request to api instagram

thanks

**Comment by desecho** [comment]
@ri7nz That's strange, I never had that problem, maybe the videos could not be deleted, I didn't try those.

**Comment by ri7nz** [comment]
@desecho   i have problem when I try to delete media(videos or photos)  and maybe you can read this code from https://github.com/mgp25/Instagram-API/blob/master/src/Request/Media.php#L49

**Pull request merged**